src/index.ts

```diff
@@ -8,6 +8,8 @@ import {
   ParticipantEvent,
   FrameEvent,
   CommentEvent,
+  ComponentLifeCycleEvent,
+  WhoIsOnlineEvent,
```

Comment:
```suggestion
  ComponentLifeCycleEvent,
  WhoIsOnlineEvent,
  CommentEvent,
```

Reply:
`CommentEvent` is right above `ComponentLifeCycleEvent` in the original code